lib/accumulate_distribute/util/gen_order_amounts.js

```diff
@@ -1,10 +1,39 @@
 'use strict'
 
 const _isFinite = require('lodash/isFinite')
-const { prepareAmount } = require('bfx-api-node-util')
 const { Config } = require('bfx-api-node-core')
+const { prepareAmount } = require('bfx-api-node-util')
+const getMinMaxDistortedAmount = require('../../util/get_min_max_distorted_amount')
+const getRandomNumberInRange = require('../../util/get_random_number_in_range')
+
 const { DUST } = Config
 
+const addRemainingToExistingOrderAmounts = (orderAmounts, opts) => {
```

Comment:
with this function we lose distortion in many cases, as per suggestion of abi, we'll remove it and go with just the warning

Reply:
removed and added info for leaving out remaining amount